dali/test/python/test_pipeline.py

```diff
@@ -424,6 +424,19 @@ def define_graph(self):
             img_chw = img_chw_test
         assert(np.sum(np.abs(img_chw - img_chw_test)) == 0)
 
+def test_none_seed():
+    batch_size = 10
+    shape = (120, 60, 3)
+    pipe = Pipeline(batch_size=batch_size, num_threads=4, device_id=0, seed=None)
```

Comment:
A useful test (if we indeed need any?) would be to build two pipelines and check that they have different output. This kind of test would possibly need a few re-runs in case we get extremely unlucky - however, if pipeline gives us exactly the same output, it would indicate a regression.

Reply:
Done